src/index.js

```diff
@@ -69,6 +82,18 @@ export default class ReactPasswordStrength extends React.Component {
       inputClasses.push('is-password-invalid');
     }
 
+    if (inputProps && inputProps.defaultClassName) {
+      inputClasses.push(inputProps.defaultClassName);
```

Comment:
I would strongly prefer that we updated `inputProps.className` to function this way instead of creating support for a new `defaultClassName` property. I think it will mean a major version change as it has potential to be a breaking change (although I think it's _very_ unlikely).

Do you mind making this update? To be clear: I think when you pass `className` to `inputProps` it should append to our component specific classNames instead of overwriting. I dont think there is much benefit to overwriting the default classes.